packages/algorand-js/src/runtime/account.ts

```diff
@@ -98,4 +99,44 @@ export class StoreAccountImpl implements StoreAccount {
       appId: appId
     });
   }
+
+  createApp (appId: number, params: SSCDeploymentFlags): CreatedApp {
+    if (this.createdApps.length === 10) {
+      throw new Error('Maximum created applications for an account is 10');
+    }
+
+    const appParams: SSCParams = {
+      'approval-program': '',
+      'clear-state-program': '',
+      creator: params.sender.addr,
+      'global-state': [],
+      'global-state-schema': { 'num-byte-slice': params.globalBytes, 'num-uint': params.globalInts },
+      'local-state-schema': { 'num-byte-slice': params.localBytes, 'num-uint': params.localInts }
+    };
+    // create new app in sender's account
+    const newApp: CreatedApp = { id: appId, params: appParams };
```

Comment:
Let's rename this to `SSC` or `App` and make it a class. This function should be a class constructor, and here, instead of `account.createApp` we should have `account.addApp`

Reply:
Done. Thanks for suggestion